src/startup.py

```diff
@@ -0,0 +1,27 @@
+from os import system
+from tagrec import TagRecognition
+
+
+"""
+Determine if a tag is found and select either leader or follower
+"""
+
+class Startup:
+
+    def __init__(self):
+        self.tag = TagRecognition()
+        
+    def choose_role(self):
+        obj_data = self.tag.detect()
+        if not obj_data:
+            system('python leader.py')
```

Comment:
This runs a shell and execute the string s a bash command. Doing it this way is highly inefficient, unreliable, and insecure. It should be able to initialize and execute the leader code in Python natively.